editor/osm_editor.cpp

```diff
@@ -41,6 +41,7 @@ using std::move, std::make_shared, std::string;
 
 namespace
 {
+// Do not change, this is tags in edits.xml
```

Comment:
We need to be able to read both tags: the old one and the new one. But only the new one should be written/saved back.